includes/class-demo-poem.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Demo poem class.
+ *
+ * @link       https://nataliesmith.dev
+ * @since      1.0.0
+ * @package    Poetwp\Includes
+ */
+
+/**
+ * Represents a demo poem with a predefined title, status, and type.
+ *
+ * The `Demo_Poem` class provides static methods to retrieve the content and custom fields for a demo poem.
+ */
+class Demo_Poem {
```

Comment:
Since we wouldn't want anyone to extend this class, consider making it final:

```suggestion
final class Demo_Poem {
```